projectDocs/issues/triage.md

```diff
@@ -143,13 +143,22 @@ An issue is triaged if it is ready to be worked on.
 New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
 Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
 A `triaged` issue should also have a [priority label](#priority).
-Generally these labels are provided by NV Access only, particularly for issues that are controversial, or priority is unclear.
 
-For controversial changes, a product decision from NV Access may be required before applying the `triaged` label.
-This can be indicated with adding the label `blocked/needs-product-decision`.
+Community members should avoid adding a `triaged` label to feature requests or other decisions potentially involving significant or controversial changes to NVDA features or functionality, which may require community input or approval from NV Access.
+The label should also be avoided for other issues that are controversial, or where the priority is unclear, such as bug fixes with an unclear solution.
```

Comment:
I think we need to make the difference between the issue reported, and the bug fix submited as a pr. Bug fixes can be controversial from an UX perspective (e.g. a pr that introduces a degradation in user experience or other significant risks).  On the other hand,  a reported issue can be unclear or not reproducible. But it is difficult in my view to define „controversial“ for a reported bug. I would rather replace „controversial“ by „unclear community support“. This is quite easy to distinguish, e.g. many comments where people exchange pro and contra arguments back and forth, and there is no clear conclusion on how a solution could look like.

Reply:
After internal discussion we'd prefer to keep the current wording. Unclear community support requires some back and forth discussion. However, we'd want to ensure controversial issues aren't triaged before that conversation has a chance to occur. 
When in doubt, if its unclear if the solution or issue seems controversial, we'd simply encourage not adding the triaged label.